src/main/java/frc/robot/commands/revolver/UnjamRevolver.java

```diff
@@ -29,6 +29,7 @@ public UnjamRevolver(Revolver revolver) {
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
+    m_revolver.resetReps();
```

Comment:
I think this reset can be handled by the subsystem itself (ie. in `set()`), allowing us to completely remove the `resetReps` variable and make that logic all internal.

@Jacob4649 what do you think of this?

Reply:
I completely agree, most if not all of the logic can be handled internally within the set method